apps/condo/domains/billing/hooks/useReceiptTableFilters.tsx

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react'
+import { ComponentType, convertToOptions, FilterComponentSize, FiltersMeta } from '@condo/domains/common/utils/filters.utils'
+import { BillingIntegrationOrganizationContext, BillingReceiptWhereInput } from '@app/condo/schema'
+import { getFilter, getStringContainsFilter } from '@condo/domains/common/utils/tables.utils'
+import get from 'lodash/get'
+import { useIntl } from '@open-condo/next/intl'
+import { BillingCategory } from '../utils/clientSchema'
+import { BillingCategory as BillingCategoryType } from '@app/condo/schema'
```

Comment:
Best to reorder imports:

1. External packages absolute imports (lodash, react)
2. Open-condo packages imports (`@open-condo`)
3. Imports from condo, but other domains (`@condo`)
4. Imports from same domain